src/main/java/com/increff/commons/sheet/AbstractTSVFile.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright (c) 2021. Increff
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.increff.commons.sheet;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVWriter;
+
+import java.io.*;
+import java.util.*;
+
+public abstract class AbstractTSVFile<T> implements IDataFile<T> {
+
+    private String newlineValue = "\n";
+    private String delimValue = "\t";
+    private String nullValue = "null";
+    private Integer maxRows = 1_00_000;
```

Comment:
we have to change max rows

Reply:
this change is not needed as max row in overwritten in imdb